docs/site/HasMany-relation.md

```diff
@@ -0,0 +1,155 @@
+---
+lang: en
+title: 'Relations'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/HasMany-relation.html
+summary:
+---
+
+## Defining a HasMany Relation
```

Comment:
I think the logical separation of the sections makes perfect sense, but wondering if we can make the title/first sentence clearer to describe the purpose... to make it easier to read/understand.

IIUC, 
- Defining a HasMany Relation - it's defining the hasMany relation at the model level using @hasMany decorator.
- Configuring a HasMany relation - configuring the relation at the repository level (something along the line).  Does it control how the data is retrieved or how the payload look like? 
- Using HasMany constrained repository in a controller - it's about the usage in controller.

Reply:
I am proposing to start with a short overview providing a high-level list of steps needed to add a relation to a LB4 app. For example:

```markdown
To add a HasMany relation to your LoopBack application, you need to perform the following steps:

 1. Add a property to your model to access related model instances.
 2. Modify the source model repository class to provide access to 
    a constrained target model repository
 3. (...)
```